src/core/containers/error-simulation/SimulateClientError.js

```diff
@@ -9,21 +9,41 @@ import './SimulateClientError.scss';
 export class SimulateClientErrorBase extends React.Component {
   constructor(props) {
     super(props);
-    this.state = { buttonClicked: false };
+
+    this.state = {
+      buttonClicked: false,
+      throwRenderError: false,
+    };
   }
 
   onClick = () => {
     this.setState({ buttonClicked: true });
     setTimeout(() => this.setState({ buttonClicked: false }), 3000);
+
     throw new Error('This is a simulated client error');
   }
 
+  throwRenderError = () => {
+    this.setState({ throwRenderError: true });
+  }
+
   render() {
     const prompt = this.state.buttonClicked ?
       'Nice! Check Sentry' : '💣 Go ahead, trigger an error';
+
+    if (this.state.throwRenderError) {
+      throw new Error('This is a simulated client render error');
+    }
+
     return (
       <div className="SimulateClientError">
-        <Button buttonType="neutral" onClick={this.onClick}>{prompt}</Button>
+        <Button buttonType="neutral" onClick={this.onClick}>
```

Comment:
While you're in here, it would be nice if the component showed a paragraph explaining why this simulation page exists and what you can do with it. Maybe try to think of something that would have helped you when you first saw it.